frozen-abi/src/abi_example.rs

```diff
@@ -490,8 +491,7 @@ impl<T: Serialize + ?Sized> AbiEnumVisitor for T {
 impl<T: Serialize + ?Sized + AbiExample> AbiEnumVisitor for T {
     default fn visit_for_abi(&self, digester: &mut AbiDigester) -> DigestResult {
         info!("AbiEnumVisitor for (default): {}", type_name::<T>());
-        T::example()
```

Comment:
instantiating `T::example()` was a bug, which was uncovered with `AbiExample for PacketFlags`. surprised this isn't spotted for almost 3.5 years..

Reply:
well, i was wrong... 35dd93725ff1caa45287ae5644fcb3bbd255b983